tests/prometheus_test.go

```diff
@@ -128,6 +133,30 @@ func testPrometheusRule(token, promQuery, value string, t *testing.T) {
 	}, 2*time.Minute, 1*time.Second).Should(BeTrue())
 }
 
+func testPrometheusRunbookURLs(k8sClient *kubernetes.Clientset) {
+	var promRule monitoringv1.PrometheusRule
+	err := k8sClient.RESTClient().Get().
+		Resource("prometheusrules").
+		Namespace(namespace).
+		Name("prometheus-hpp-rules").
+		Do(context.TODO()).Into(&promRule)
+	Expect(err).ToNot(HaveOccurred())
+	for _, group := range promRule.Spec.Groups {
+		if group.Name == "hpp.rules" {
+			for _, rule := range group.Rules {
```

Comment:
Maybe we want assertions here that Spec.Groups/Rules is not empty

Reply:
Done

Reply:
Think we also need to check .Groups (sorry for not being clear)